pkg/resource/obtenantbackuppolicy_task.go

```diff
@@ -193,8 +193,16 @@ func (m *ObTenantBackupPolicyManager) StopBackup() error {
 	if err != nil {
 		return err
 	}
-
-	_ = con.DisableArchiveLogForTenant()
```

Comment:
Check the status of the tenant in database to decide wether to call DisableArchiveLogForTenant()